### PR TITLE
Prefetch messages with the new iOS 10 API

### PR DESCRIPTION
**Pull request by mikeger**
Implemented `UITableViewDataSourcePrefetching` for conversation. Not sure if it would make the scrolling faster, the PR is more to try it and check the visual difference in the performance.

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Conversation/Content/ConversationContentViewController.m [hunk 1]
```diff
@@ -689,6 +692,17 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
...
 }
 
+- (void)tableView:(UITableView *)tableView prefetchRowsAtIndexPaths:(NSArray<NSIndexPath *> *)indexPaths
+{
+    NSArray<NSIndexPath *> *sortedIndexPaths = [indexPaths sortedArrayUsingSelector:@selector(row)];
+    
+    NSIndexPath* latestIndexPath = sortedIndexPaths.lastObject;
+    
+    if (latestIndexPath.row + 10 > (int)self.messageWindow.messages.count) {
+        [self expandMessageWindowUp];
```
Comment (daehn) [inline Wire-iOS/Sources/UserInterface/Conversation/Content/ConversationContentViewController.m]: Potentially we could also initiate image downloads etc. here

Comment (mikeger) [inline Wire-iOS/Sources/UserInterface/Conversation/Content/ConversationContentViewController.m]: Good point.

**Comment by mikeger** [comment]
@daehn updated

**Comment by mikeger** [comment]
@daehn updated nr.2

**Pull request merged**